commit fd5fe4a8482cbc4467cabf29e70f8efe74cbb89f (refs/changes/31/2331/4)
Author: Tim Kordas <tim.kordas@joyent.com>
Date:   2017-09-20T16:45:11+00:00 (2 years, 1 month ago)
    
    MANTA-3364 'Loadbalancer has no backends' probe is missing tooling to actually run

diff --git a/package.json b/package.json
index 558f979..068a673 100644
--- a/package.json
+++ b/package.json
@@ -11,6 +11,7 @@
         "dashdash": "1.4.0",
         "forkexec": "1.1.0",
         "jsprim": "1.4.0",
+        "haproxy-stat" : "0.1.0",
         "node-uuid": "1.4.1",
         "once": "1.3.0",
         "vasync": "2.0.0",
